app/build.gradle.kts

```diff
@@ -141,6 +141,7 @@ dependencies {
     implementation("androidx.navigation:navigation-fragment-ktx:${Versions.ANDROIDX_NAVIGATION_VERSION}")
     implementation("androidx.navigation:navigation-runtime-ktx:${Versions.ANDROIDX_NAVIGATION_VERSION}")
     implementation("androidx.navigation:navigation-ui-ktx:${Versions.ANDROIDX_NAVIGATION_VERSION}")
```

Comment:
Seems we don't use this dependency anymore. If so, we can remove it safely.

Reply:
@hoangnguyen92dn Yep, I think it better to create an issue to remove all of the dependencies of `navigation` because there're many files that need to be removed, e.g. (`nav_graph_main,` `BaseNavigator,` `NavigationEvent`) 🤔

Reply:
@Wadeewee 👌 I will update on my integration task to remove all of them.